src/vumi2/routers.py

```diff
@@ -15,10 +16,15 @@
 logger = getLogger(__name__)
 
 
+class ToAddressMapping(TypedDict):
```

Comment:
Can we use an `attrs` class here for consistency, rather? (We already have the necessary machinery to deserialise those.)